apps/meeting/src/components/DeviceSelection/CameraDevices/index.tsx

```diff
@@ -25,9 +28,11 @@ const CameraDevices = () => {
       <StyledInputGroup>
         <QualitySelection />
       </StyledInputGroup>
+      { blurOption !== BlurValues.blurDisabled ?
       <StyledInputGroup>
         <BackgroundBlurCheckbox />
-      </StyledInputGroup>
+      </StyledInputGroup> : ''
```

Comment:
Minor can fix later but Instead of "" can we just render like below (dont know what "" shows like 😅 ) : 
```
{ blurOption !== BlurValues.blurDisabled &&
      <StyledInputGroup>
        <BackgroundBlurCheckbox />
      </StyledInputGroup>}
```